README.md

```diff
@@ -1,2 +1,3 @@
 # SoMeApp_me-and-my-friends
-"Me &amp; My Friends" is a social media website where the user has more control over customization. set their own theme that appears on their banner when posting. This project is a project made das a school project.  
+
```

Comment:
How do I run the project? How do I add to it? Which technologies does it use?

Reply:
[RESOLVED] readme now contains detailed instructions on how to clone, install and run the app. it also includes other relevant info aobut the project.